samples/nrf5340/multiprotocol_rpmsg/prj.conf

```diff
@@ -23,5 +23,5 @@ CONFIG_DEBUG_INFO=y
 CONFIG_EXCEPTION_STACK_TRACE=y
 
 CONFIG_NRF_802154_SER_RADIO=y
-CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=2
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=3
```

Comment:
Is this part related to `https://github.com/nrfconnect/sdk-nrf/pull/9908` ? Will there be a need to increase the channel count further, or will 3 be ok?

Reply:
There will be no need to increase the parameter further. Yes pull/9908 it is somehow related - contains the same commit as here. It was opened for another reason, when it was not yet known when the work would resume here. If we manage to merge this first, then pull/9908 can be closed.

Reply:
pull/9908 was merged earlier. I removed the commit with the modifications you asked about from here